tools/kbump/go.mod

```diff
@@ -0,0 +1,62 @@
+module github.com/aws/karpenter/tools/kbump
+
+go 1.21.1
```

Comment:
As a general practice, we're only requiring the minor version, not the patch version